testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -222,7 +226,7 @@ Map edxEnterprise = [
     repoName: 'edx-enterprise',
     targetBranch: "master",
     pythonVersion: '3.5',
-    cronValue: cronOffHoursBusinessWeekday,
+    cronValue: cronOffHoursBusinessWeekdayLahore,
```

Comment:
Maybe @jmbowman needs to speak to these owned by other teams. I’m not sure what was agreed upon. Feels like either you should be added to githubTeamReviewers, or you shouldn’t be adjusting the schedule, but this isn’t really my decision.

Reply:
Yes, you are right, in case of this repo, reviewer team is `business-enterprise-team` but email were being sent to `arch-bom` previously, so I changed that to arbi-bom, imo I should revert this change, but still @jmbowman can guide better.

Reply:
I suspect that email was a copy/paste error; @georgebabey , what email address should receive notifications when an edx-enterprise "make upgrade" job fails to get far enough to even create a PR?

Reply:
I've reverted cronValue for this repo.